docs/managed-victoriametrics/quickstart.md

```diff
@@ -11,179 +11,265 @@ aliases:
+  * Cluster - for highly available and multi-tenant deployments;
+* `Region` – AWS region where deployment will run;
 * Desired `storage capacity` for storing metrics (you always can expand disk size later);
 * `Retention` period for stored metrics.
-* `Size` of your deployment
+* `Size` of your deployment [based on your needs](https://docs.victoriametrics.com/guides/understand-your-setup-size.html)
 
 <p>
-  <img src="quickstart_deployment-create.png" width="800">
+  <img src="create_deployment_form.png" width="800">
 </p>
 
-When all parameters are entered, click on the `Create` button, and deployment will be created
+When all parameters are configured, click on the `Create` button, and deployment will be created.
 
 Once created, deployment will remain for a short period of time in `PROVISIONING` status 
 while the hardware spins-up, just wait for a couple of minutes and reload the page. 
-You'll also be notified via email once provisioning is finished:
+You'll also be notified via email once your deployment is ready to use:
 
 <p>
-  <img src="quickstart_deployment-created.png" width="800">
+  <img src="create_deployment_created.png" width="800">
 </p>
 
 <p>
-  <img src="quickstart_deployments-running.png" width="800">
+  <img src="create_deployment_active_email.png" width="800">
 </p>
 
-## Deployment access
+## Start writing and reading data
+
+After transition from `PROVISIONING` to `RUNNING` state, Managed VictoriaMetrics deployment is fully operational 
+and is ready to accept write and read requests. 
 
-After transition from `PROVISIONING` to `RUNNING` state, VictoriaMetrics is fully operational 
-and ready to accept write or read requests. But first, click on deployment name to get the access token:
+Click on deployment name and navigate to the Access tab to get the access token:
 
 <p>
-  <img src="quickstart_deployment-access-token.png" width="800">
+  <img src="deployment_access.png" width="800">
 </p>
 
 Access tokens are used in token-based authentication to allow an application to access the VictoriaMetrics API. 
 Supported token types are `Read-Only`, `Write-Only` and `Read-Write`. Click on token created by default 
 to see usage examples:
 
 <p>
-  <img src="quickstart_read-token.png" width="800">
+  <img src="deployment_access_write_example.png" width="800">
 </p>
 
 <p>
-  <img src="quickstart_write-token.png" width="800">
+  <img src="deployment_access_read_example.png" width="800">
 </p>
 
-Follow usage example in order to configure access to VictoriaMetrics for your Prometheus, 
+Follow usage examples in order to configure access to VictoriaMetrics for your Prometheus, 
 Grafana or any other software.
 
-## Modifying deployment
+## Modifying an existing deployment
 
-Remember, you always can add, remove or modify existing deployment by changing their size or any parameters on the 
-update form.
+Remember, you can always add, remove or modify existing deployment by changing its size or any parameters on the 
```

Comment:
please mention that downgrade for cluster is currently not available (if this is a right place for that)

Reply:
added this warning in the text below